client/src/test-setup.ts

```diff
@@ -4,4 +4,9 @@ import '@angular/compiler';
 import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
 import { getTestBed } from '@angular/core/testing';
 
-getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+try {
```

Comment:
question: could you try to rebase to the newest version and see if your error still occurs?